Build WebUI on GitHub

akuskis opened the pull request.

embalex commented on deployment/webui/build-webui.sh:
```diff
@@ -9,7 +9,7 @@ cd "$SCRIPT_PATH"
 BUILD_PATH=/app
 WEB_STATIC_IMAGE_NAME=test
-NGINX_INAGE_NAME=embalex/webui
+NGINX_INAGE_NAME=dndx/webui
```
О! Опечаточку нашел. NGINX_I**N**AGE_NAME

akuskis replied:
Yep, there will be a new version with fixed typo

embalex commented:
I saw to the folder structure and I have a little bit misunderstanding. At the start of the project there are 3 folders are planning: frontend (client folder), about environments and server structures (deployment  folder) and all services (server folder). One more folder is stay out - servers-script (scripts should be moving to deploy or server later). Now I am confusing about services. Is it server folder? Anyway it contains webui, and web ui is not services, its about deployment environment imho.
So lets discuss it.

akuskis commented:
> I saw to the folder structure and I have a little bit misunderstanding. At the start of the project there are 3 folders are planning: frontend (client folder), about environments and server structures (deployment folder) and all services (server folder). One more folder is stay out - servers-script (scripts should be moving to deploy or server later). Now I am confusing about services. Is it server folder? Anyway it contains webui, and web ui is not services, its about deployment environment imho. So lets discuss it.

Sure. Before we got "client" (which also build itself into container and was ready for deployment) and "server" (stuff which should be able to provide their containers too). Also "deployment" folder which contains deployment configuration (k8s related files).
Later "client" part related to container building had leaked into the "deployment". So deployment also means building only a part of the sources, pushing them into the Dockerhub and pulling back.
____
In this case I suggest to keep in the "deployment" folder only k8s related stuff. So I've renamed "server" to "services" and added new "service" which will serve webui static.

deployment > k8s setup
services > services like "dice roller" and "webui"
client > react project (what if we will move client into services/webui/client)?
